### ISSUE TITLE
Suddenly Cannot Access with Admin role (login failed)

### ISSUE THREAD

**Issue by joko2020** [dotproject/dotProject#143]
I have successfully installed dotproject version 2.2.0 and have succeeded in creating several users including users with admin access. The problem suddenly comes after the application has been running for more than three days, I cannot log in with the admin user (I have changed the initial password) and users with admin access, but other users with project work access can access the dot project.
what should i do to fix this ??

**Comment by ipeevski** [comment]
Did you try "Help! I've forgotten my password!" link on the login screen?
Another option would be to update the database directly ...

**Comment by joko2020** [comment]
I've tried, but did not solved ...I always have 3 backup user administrator account with different Username but have exactly same password...I tried 3 all of my administrator account but still cannot logged in...but other user (not Administrator Role) working fine.
after googling many advise told that DB must be repair... and I Tried to Re deploy dotproject with same mysql db.. problem solved.. and after I check user session there is few row blank value before latest user log in.. i don't know why but I think it is connection problem between dotproject with mysql...

**Comment by ipeevski** [comment]
So your issue is resolved for now?

**Comment by joko2020** [comment]
Yes, but i would like to find why this can happen.. if i found it i would like to posting it here, can I??

**Comment by ipeevski** [comment]
Of course :)
